### DISCUSSION TITLE
kpanda/best-practice/etcd-backup/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1177] [Announcements]
# kpanda/best-practice/etcd-backup/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/best-practice/etcd-backup/

**Comment by windsonsea** [comment]
查看 etcd 集群状态 这一节的代码段都乱了，mark 一下

**Reply by Michelle951** [reply]
fixed
